code/modules/mob/living/simple_animal/abnormality/waw/express_train.dm

```diff
@@ -171,6 +155,7 @@
 				src.segments -= seg
 			else
 				seg.forceMove(get_step(seg, seg.dir))
+				seg.forceMove(get_step(seg, seg.dir))
```

Comment:
Can't you just change delay on the timer above to 0.5?

Reply:
... The timer is one tick, is that not the smallest possible unit of time??